BTCPayServer.Client/Models/CreateInvoiceRequest.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using BTCPayServer.JsonConverters;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace BTCPayServer.Client.Models
+{
+    public class CreateInvoiceRequest
+    {
+        [JsonProperty(ItemConverterType = typeof(DecimalDoubleStringJsonConverter))]
+        public decimal Amount
+        {
+            get;
+            set;
+        }
```

Comment:
Formatting: The props should all be consistenly formatted, hence:

```suggestion
        public decimal Amount { get; set; }
```